test/integration/default/files/_mapdata/centos-6.yaml

```diff
@@ -0,0 +1,34 @@
+# yamllint disable rule:indentation rule:line-length
```

Comment:
This file will probably need updating as I fix the `centos6` suite (which really should be renamed as well).

Reply:
Proposed fix: https://github.com/saltstack-formulas/template-formula/pull/203#discussion_r481298806.

Reply:
Fix provided: #204.